requirements.txt

```diff
@@ -1,24 +1,102 @@
-altair==4.2.2
-requests<2.32.0
-nowcasting_datamodel==1.5.42
-pvsite-datamodel==1.0.45
-numpy==1.24.1
-pandas==1.5.3
-plotly==5.10.0
-psycopg2-binary==2.9.5
-SQLAlchemy==2.0.29
-streamlit==1.27.2
-testcontainers==3.2.0
-uvicorn==0.23.2
-geopandas==0.14.2
-garrett-streamlit-auth0==0.9
-pydantic==2.5.3
-xarray==2023.5.0
-fsspec==2024.2.0
-s3fs==2024.2.0
-ocf_blosc2==0.0.4
-zarr==2.14.2
-elexonpy
-fiona==1.9.6
-psycopg2-binary
-herbie-data
\ No newline at end of file
+aiobotocore==2.17.0
+aiohappyeyeballs==2.4.4
+aiohttp==3.11.11
+aioitertools==0.12.0
+aiosignal==1.3.2
+alembic==1.14.0
+altair==5.5.0   # previously was 4.1.0
+annotated-types==0.7.0
+anyio==4.8.0
+asciitree==0.3.3
+attrs==24.3.0
+blinker==1.9.0
+blosc2==3.0.0
+botocore==1.35.93
+cachetools==5.5.0
+certifi==2024.12.14
+charset-normalizer==3.4.1
+click==8.1.8
+click-plugins==1.1.1
+cligj==0.7.2
+colorama==0.4.6
+docker==7.1.0
+ecdsa==0.19.0
+elexonpy==1.0.15
+fasteners==0.19
+fiona==1.10.1  # previously was 1.9.6
+freezegun==1.5.1
+frozenlist==1.5.0
+fsspec==2024.12.0 # previously was 2024.2.0
+garrett-streamlit-auth0==0.9 
+geopandas==1.0.1 # previously was 0.14.2
+gitdb==4.0.12
+GitPython==3.1.44
+greenlet==3.1.1
+h11==0.14.0
+httpcore==1.0.7
+httpx==0.28.1
+idna==3.10
+Jinja2==3.1.5
+jmespath==1.0.1
+jsonschema==4.23.0
+jsonschema-specifications==2024.10.1
+Mako==1.3.8
+markdown-it-py==3.0.0
+MarkupSafe==3.0.2
+mdurl==0.1.2
+msgpack==1.1.0
+multidict==6.1.0
+narwhals==1.21.1
+ndindex==1.9.2
+nowcasting_datamodel==1.5.56  # previously was 1.5.55
+numcodecs==0.13.1
+numexpr==2.10.2
+numpy==2.0.0  # previously was 1.24.1
+ocf_blosc2==0.0.11 # previously was 0.0.4
+packaging==24.2
+pandas==2.2.3 # previously was 1.5.3
+pillow==11.1.0
+plotly==5.24.1 # previously was 5.10.0
+propcache==0.2.1
+protobuf==5.29.2
+psycopg2-binary==2.9.10 # previously was 2.9.5
+pvsite-datamodel==1.0.47  # previously was 1.0.45
+py-cpuinfo==9.0.0
+pyarrow==18.1.0
+pyasn1==0.6.1
+pydantic==2.5.3 # previously was 2.5.3
+pydantic_core==2.14.6
+pydeck==0.9.1
+Pygments==2.19.1
+pyogrio==0.10.0
+pyproj==3.7.0
+python-dateutil==2.9.0.post0
+python-dotenv==1.0.1
+python-jose==3.3.0
+referencing==0.35.1
+requests==2.32.3 # previously was 2.26.0
+requests-mock==1.12.1
+rich==13.9.4
+rpds-py==0.22.3
+rsa==4.9
+s3fs==2024.12.0 # previously was 2024.2.0
+shapely==2.0.6
+six==1.17.0
+smmap==5.0.2
+sniffio==1.3.1
+SQLAlchemy==2.0.36 # previously was 2.0.29
+streamlit==1.41.1 # previously was 1.27.2
+structlog==24.4.0
+tenacity==9.0.0
+testcontainers==4.9.0 # previously was 3.2.0
+toml==0.10.2
+tornado==6.4.2
+typing_extensions==4.12.2
+tzdata==2024.2
+urllib3==2.3.0
+uvicorn==0.34.0  # previously was 0.23.2
+watchdog==6.0.0
+wrapt==1.17.0
+xarray==2025.1.0 # previously was 2023.5.0
+yarl==1.18.3
+zarr==2.18.4  # previously was 2.14.2
```

Comment:
sorry, i meant only include requirements that were there before, not all the subrequirements. I understand you mught need to add one or too.

Please remove `previously was x.y.z` as its in the git history